src/CMakeLists.txt

```diff
@@ -26,10 +25,13 @@ set(source
 )
 add_library(libHPWHsim ${source} ${headers})
 
+target_include_directories(libHPWHsim PUBLIC ${PROJECT_BINARY_DIR}/src ${PROJECT_SOURCE_DIR}/src ${PROJECT_SOURCE_DIR}/vendor/btwxt/src)
```

Comment:
Should not include the btwxt directory. That is added by linking to btwxt through its PUBLIC include directory.

Reply:
Fixed